asylum-connect/control-panel
# Feature/1197054159713390/social media

## Description

- adds social media section to org page

## Asana ticket: https://app.asana.com/0/1132189118126148/1197054159713390/f

## PR Checklist



- [x] If your PR is not a hotfix, is it targeted for `dev`? If your PR is a hotfix, is it targeted for `main`?

## How to Test

1. navigate to the control panel
2. Log in
3. View an organization
4. scroll down to "Social Media" section. verify it exists
5. Click New social media profile
6. Fill out form and save changes. Verify that new data is displayed correctly in Social Media section
7. Edit social media data. Verify that data has been updated accordingly 
8. Delete social media profile. Verify that data is no longer being displayed in Social Media section

### **Best tested locally, or after corresponding catalog api PR has been merged as this depends on those  changes to work correctly**

![image](https://user-images.githubusercontent.com/7406914/122586582-da9f2a00-d02a-11eb-91a1-1fccb6ac0b13.png)

![image](https://user-images.githubusercontent.com/7406914/122586651-ee4a9080-d02a-11eb-9314-3cd8d4b86935.png)

![image](https://user-images.githubusercontent.com/7406914/122586750-09b59b80-d02b-11eb-83df-73264e45f9db.png)

![image](https://user-images.githubusercontent.com/7406914/122586774-13d79a00-d02b-11eb-8c3b-7cb46f36dece.png)

![image](https://user-images.githubusercontent.com/7406914/122586815-1c2fd500-d02b-11eb-8302-3b2957e46a45.png)
